applications/blackhole/src/blackhole_data_emitter.erl

```diff
@@ -21,7 +21,7 @@ event(Binding, RK, Name, Data) ->
           ,{<<"subscription_key">>, SubscriptionKey}
           ,{<<"name">>, Name}
           ,{<<"routing_key">>, RK}
-          ,{<<"data">>, Data}
+          ,{<<"data">>, kz_json:delete_key(<<"amqp_broker">>, Data)}
```

Comment:
@remcovanvugt why is `amqp_broker` here in the first place ?

Reply:
@lazedo I was surprised to see it there, I found this on a 4.1-53 clean install on which only a data migration from 3.22 was performed. It's not there on 3.22. 

It happens on a CHANNEL_CREATE event, I haven't check with the other events. The auth token does not belong to superduper admin. The rest of the emitted data looks fine, only channel related data..

Reply:
@remcovanvugt we can't recreate this, maybe something particular to your setup or the data you had in 3.22

Reply:
this is federated related as we talked in the community call.